hoverxref-intersphinx.rst

```diff
@@ -0,0 +1,108 @@
+.. post:: June 10, 2021
+   :tags: czi, grant, czi-grant, embed, sphinx-hoverxref
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      New release of sphinx-hoverxref, our Sphinx extension to add tooltips to cross references,
+      with support for intersphinx.
+
+New release of sphinx-hoverxref with support for intersphinx
+============================================================
+
+We have released version 0.6b1 of `sphinx-hoverxref`_,
+a Sphinx extension that shows a floating window
+that embeds the content of a cross-reference.
+
+This extension is an essential part
+of the work we are doing to improve interoperability of documentation in general,
+and scientific documentation in particular,
+thanks to the `CZI grant </czi-grant-announcement>`_ we received last year.
+
+.. figure:: img/sphinx-hoverxref-intersphinx.gif
+   :width: 100%
+   :align: center
+   :alt: sphinx-hoverxref displaying a modal window embedding an external project
+   :target: _images/sphinx-hoverxref-intersphinx.gif
```

Comment:
Does it make sense to change this GIF with a real-life example that we can link to the project as well? This was just an example that I record to show to the team, but it doesn't exist.

Also, this particular case (intersphinx feature) may need an extra explanation since it's not easy to realize that the content displayed in the modal comes from a completely different project.

Reply:
Oh, I see that you are introducing intersphinx later on in the text. So, we could add the GIF in that section with an explanation.

Here, in the beginning, we could just add the most basic case, which is the small "tooltip" style to reference an internal link.

Reply:
Fair enough! I can switch the order of the images. I originally put the animated GIF first because perhaps it captivates the reader attention :)

Reply:
The first one could still be a GIF with the small tooltip example 😉

Reply:
Swapped the images. If you still think we should record a new animation, let me know.